plugins/plugin-java/che-plugin-java-ext-lang-server/src/main/java/org/eclipse/che/plugin/java/server/rest/recommend/similarity/Similarity.java

```diff
@@ -0,0 +1,733 @@
+package org.eclipse.che.plugin.java.server.rest.recommend.similarity;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import org.eclipse.che.plugin.java.server.rest.recommend.common.Config;
+import org.eclipse.che.plugin.java.server.rest.recommend.model.DataModel;
+import org.eclipse.che.plugin.java.server.rest.recommend.model.ResultModel;
+
+public class Similarity {
```

Comment:
is this class you've made for 100%? Can you use something similar from Lucene?
javadoc, test?